core/src/test/scala/pureconfig/ConfigConvertChecks.scala

```diff
@@ -23,8 +26,8 @@ trait ConfigConvertChecks { this: FlatSpec with Matchers with GeneratorDrivenPro
    * because, for instance, multiple representation of `t: T` are possible. Use [[checkRead()]] for those
    * representations.
    */
-  def checkArbitrary[T](implicit cc: ConfigConvert[T], arb: Arbitrary[T], tag: ClassTag[T], equality: Equality[T]): Unit =
-    it should s"read an arbitrary ${tag.runtimeClass.getCanonicalName}" in forAll {
+  def checkArbitrary[T](implicit cc: ConfigConvert[T], arb: Arbitrary[T], tpe: TypeTag[T], equality: Equality[T]): Unit =
```

Comment:
Good idea to use `TypeTag`. We should switch to `TypeTag` when we just need the type name.